core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -87,6 +87,8 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_SQL_HASH_JOIN_LIGHT_VALUE_MAX_PAGES("cairo.sql.hash.join.light.value.max.pages"),
     CAIRO_SQL_ASOF_JOIN_LOOKAHEAD("cairo.sql.asof.join.lookahead"),
     CAIRO_SQL_ASOF_JOIN_FAST("cairo.sql.asof.join.fast"),
+    CAIRO_SQL_ASOF_JOIN_SHORT_CIRCUIT_CACHE_CAPACITY("cairo.sql.asof.join.short.circuit.cache.capacity"),
```

Comment:
nit: might be a good idea to add dynamic (hot reloadable) props to the list in `DynamicPropServerConfiguration`.